pages/application.tsx

```diff
@@ -83,185 +63,190 @@ const Application: NextPage = () => {
-									pre- and post-event informational e-mails, and occasional messages about hackathons in-line with the MLH Privacy Policy. I
-									further I agree to the terms of both the
-									<a
-										href="https://github.com/MLH/mlh-policies/tree/master/prize-terms-and-conditions"
-										target="_blank"
-										rel="noopener noreferrer">
-										{' '}
-										MLH Contest Terms and Conditions
-									</a>{' '}
-									and the
-									<a href="https://mlh.io/privacy" target="_blank" rel="noopener noreferrer">
-										MLH Privacy Policy.
-									</a>
-								</>
+					<Stepper
+						active={step}
+						onStepClick={(step: number) => {
+							if (!form.validate().hasErrors) {
+								setStep(step);
 							}
-							{...form.getInputProps('dataAgreement')}
-						/>
-						<Checkbox
-							label="I authorize MLH to send me pre- and post-event informational emails, which contain free credit and opportunities from their partners."
-							{...form.getInputProps('mlhAgreement')}
-						/>
+						}}>
+						<Stepper.Step label="Personal Info">
+							<Box sx={{ maxWidth: 600 }} mx="auto">
+								<Stack>
+									<TextInput required label="First Name" {...form.getInputProps('firstName')} />
+									<TextInput required label="Last Name" {...form.getInputProps('lastName')} />
+									<TextInput required label="Email" placeholder="your@email.com" {...form.getInputProps('email')} />
+									<TextInput label="Phone Number" {...form.getInputProps('phoneNumber')} />
+									<NativeSelect required label="School" {...form.getInputProps('school')} data={['MST', 'Mizzou', 'Other School']} />
+									<NativeSelect
+										required
+										label="Graduation Year"
+										{...form.getInputProps('graduationYear')}
+										data={[
+											{ value: 'Graduate', label: 'Graduate Student' },
+											{ value: '2023', label: '2023' },
+											{ value: '2024', label: '2024' },
+											{ value: '2025', label: '2025' },
+											{ value: '2026', label: '2026' },
+											{ value: '2027', label: '2027' },
+											{ value: '2028', label: '2028' },
+											{ value: 'High School', label: 'High School' }
+										]}
+									/>
+									<Group position="right" mt="md">
+										<Button
+											onClick={() => {
+												if (!form.validate().hasErrors) {
+													setStep(step + 1);
+												}
+											}}>
+											Next
+										</Button>
+									</Group>
+								</Stack>
+							</Box>
+						</Stepper.Step>
+						<Stepper.Step label="Prior Experience">
+							<Box sx={{ maxWidth: 600 }} mx="auto">
+								<Stack>
+									<NativeSelect
+										required
+										label="T-Shirt Size"
+										{...form.getInputProps('shirtSize')}
+										data={['XS', 'S', 'M', 'L', 'XL', 'XXL']}
+									/>
+									<MultiSelect
+										label="Dietary Restrictions"
```

Comment:
Food options should be required, and also I want to try this feature.
```suggestion
required
label="Dietary Restrictions"
```

Reply:
I don't know how it's gonna format the suggestion, full disclosure

Reply:
i think it would be weird UX if we forced them to pick an option even without dietary restrictions, cuz the vast majority of people won't have dietary restrictions

Reply:
Yeah, I think so too. Though I think if we want to have it not required as it currently is, we'd have to remove none as an option within dietary restriction. It doesn't really make sense if it's chosen along with the other options, and the absence of a selection means none already.